src/content/advanced/observability/visualization/_index.md

```diff
@@ -0,0 +1,13 @@
+---
+linkTitle: Visualization
+title: Visualization
+description: Guide explaining how to interact with our installation level grafana (how to access grafana, how to explore logs and metrics).
```

Comment:
I'd suggest using a different description than the `General` page

Reply:
I agree but i'm not sure what to use 😂

Reply:
I'm open to any suggestions